includes/customer-functions.php

```diff
@@ -217,6 +217,11 @@ function edd_get_customers( $args = array() ) {
 		'number' => 30
 	) );
 
+	if ( -1 == $args['number'] ) {
+		_doing_it_wrong( __FUNCTION__, esc_html__( 'Do not use -1 to retrieve all results.', 'easy-digital-downloads' ), '3.0' );
+		$args['number'] = 9999999;
```

Comment:
```suggestion
		$r['number'] = 9999999;
```

To match with the above.